src/Middleware/Session/src/SessionMiddleware.cs

```diff
@@ -108,7 +108,7 @@ public async Task Invoke(HttpContext context)
             }
             finally
             {
-                context.Features.Set<ISessionFeature>(null);
+                context.Features.Set<ISessionFeature?>(null);
```

Comment:
❔ Can you link me to the definition of `Set<T>`?

Reply:
https://github.com/dotnet/aspnetcore/blob/15f341f8ee556fa0c2825cdddfe59a88b35a87e2/src/Http/Http.Features/src/FeatureCollection.cs#L101-L104